completions/tmatrix-completion.tcsh

```diff
@@ -0,0 +1,54 @@
+# Copyright (C) 2018-2019 Miloš Stojanović
+#
+# SPDX-License-Identifier: GPL-2.0-only
+
```

Comment:
Add this here:
```
# To use this completion script:
#    1) Copy this file to ${HOME}.
#    2) Add the following line to your .tcshrc/.cshrc:
#        source ~/.tmatrix-completion.tcsh
```

You can also merge the commits.

Reply:
The `source` command filename has a leading dot. We should note it, for example:

```
#    1) Copy this file to ${HOME}.
#        cp completions/tmatrix-completion.tcsh ${HOME}/.tmatrix-completion.tcsh
```

Though it's too verbose :(

Reply:
Yea, it should be noted. How about this:
```
#    1) Copy this file to ${HOME}:
#        cp tmatrix-completion.tcsh ~/.tmatrix-completion.tcsh
```